Added sections for Weaviate and Qdrant

padma-ietf opened the pull request.

padma-ietf committed:
Added sections for Weaviate and Qdrant

dblock commented on _posts/2023/2023-06-06-getting-started-with-vector-dbs-in-python.md:
```diff
@@ -376,6 +376,151 @@ ENDPOINT=http://localhost:8080 CONFIG_ENDPOINT=http://localhost:19071 poetry run
 {% endhighlight %}
 
+### Weaviate
+Weaviate is a vector search engine specifically designed for natural language and numerical data. It uses contextualized embeddings in data objects to understand semantic similarity. Currently, it supports only Hierarchical Navigable Small World(HNSW) indexing and is more costly on building data to indexes however has a fast query time and high scalability. Weaviate is open-source, easy to use, f ...
```
Add a space before `(HNSW)` and an oxford comma before ", and is more costly" and ", and has a CLA". Make "However, it claims fast query time ..." a sentence.

dblock replied:
Add a an empty line after the title. Link the first Weviate in the sentence.

dblock commented on _posts/2023/2023-06-06-getting-started-with-vector-dbs-in-python.md:
```diff
@@ -376,6 +376,151 @@ ENDPOINT=http://localhost:8080 CONFIG_ENDPOINT=http://localhost:19071 poetry run
+### Weaviate
+Weaviate is a vector search engine specifically designed for natural language and numerical data. It uses contextualized embeddings in data objects to understand semantic similarity. Currently, it supports only Hierarchical Navigable Small World(HNSW) indexing and is more costly on building data to indexes however has a fast query time and high scalability. Weaviate is open-source, easy to use, f ...
+
+After you sign up at Weaviate Cloud Services WCS, and create a new free tier Weaviate Cluster with authentication. Note your cluster URL and API key (optional). The endpoint will have the following format https://myindex.weaviate.network.
```
Firs sentence reads strange, remove "and".

dblock commented on _posts/2023/2023-06-06-getting-started-with-vector-dbs-in-python.md:
```diff
@@ -376,6 +376,151 @@ ENDPOINT=http://localhost:8080 CONFIG_ENDPOINT=http://localhost:19071 poetry run
+    client.post(urljoin(endpoint, "/v1/batch/objects"), json={"objects": objects}, headers=headers)
+{% endhighlight %}
+
+The search is pretty straight forward. Weaviate also has a GraphQl interface online.
```
straightforward is one word.

GraphQL is double capital QL

remove "online".

dblock commented on _posts/2023/2023-06-06-getting-started-with-vector-dbs-in-python.md:
```diff
@@ -376,6 +376,151 @@ ENDPOINT=http://localhost:8080 CONFIG_ENDPOINT=http://localhost:19071 poetry run
+
+
+### Qdrant
+Qdrant is similarity vector search engine designed for a wide range of applications, including recommendation systems, image search, and natural language processing. It is open-source, scalable and allows dynamic updates to the index. It is particularly suitable for scenarios where the vector data is constantly evolving and vectors may be modified without interrupting the search functionality. Qu ...
```
Add a line, link Qdrant.

You say open-source and licensed under APLv2, it's the same thing. Would just leave one.

dblock commented on _posts/2023/2023-06-06-getting-started-with-vector-dbs-in-python.md:
```diff
@@ -376,6 +376,151 @@ ENDPOINT=http://localhost:8080 CONFIG_ENDPOINT=http://localhost:19071 poetry run
+
+Qdrant is built upon a concept of indexes, where vectors are organized and stored in "collections" for quick retrieval. Currently it only supports HNSW (Hierarchical Navigable Small World) as vector index.
+
+After you sign up at Weaviate Cloud Services WCS, and create a new free tier Weaviate Cluster with authentication. Note your cluster URL and API key. The endpoint will have the following format https://my-cluster.cloud.qdrant.io:6333/.
```
remove "and"

dblock reviewed (changes requested):
Almost perfect, have some language nits, please? Apologies for my OCD :)

padma-ietf committed:
Addressed review comments
Fixed a few more issues in Qdrant section
Removed reference to Weaviate and Qdrant in the list of examples
